internal/config/mount_config.go

```diff
@@ -35,9 +35,16 @@ const (
 	// The maximum multiple of seconds representable by time.Duration.
 	MaxSupportedTtlInSeconds int64 = int64(math.MaxInt64 / int64(time.Second))
 
-	// DefaultTypeCacheMaxEntries is the default vlaue of type-cache-max-entries.
-	// The value is set high at 1,048,476 (2^20) as it is not supposed to be set by users.
-	DefaultTypeCacheMaxEntries int = 2 << 20
+	// DefaultTypeCacheMaxSizeMB is the default vlaue of type-cache max-size for every directory in MiBs.
+	// The value is set at the size needed for about 21k type-cache entries,
+	// each of which is about 200 bytes in size.
+	DefaultTypeCacheMaxSizeMB int = 4
+
+	// AverageTypeCacheEntrySize is for internal testing
```

Comment:
"is for internal testing" -> "internal size calculations"

Reply:
done.